src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -453,10 +453,12 @@ export class TerminalService implements ITerminalService {
 
 	@debounce(500)
 	private _updateLocalState(): void {
-		const state: ITerminalsLayoutInfoById = {
-			tabs: this.terminalTabs.map(t => t.getLayoutInfo(t === this.getActiveTab()))
-		};
-		this._localTerminalService!.setTerminalLayoutInfo(state);
+		if (!!this._localTerminalService) {
+			const state: ITerminalsLayoutInfoById = {
+				tabs: this.terminalTabs.map(t => t.getLayoutInfo(t === this.getActiveTab()))
+			};
+			this._localTerminalService.setTerminalLayoutInfo(state);
+		}
```

Comment:
We might want to switch this back to what it was. Right now `_updateLocalState` should only ever be called when `_localTerminalService` is set, we probably want this to throw to draw attention to something being wrong.